code/__defines/color.dm

```diff
@@ -1,3 +1,86 @@
+#define COLOR_NAVY_BLUE        "#000080"
+#define COLOR_GREEN            "#008000"
+#define COLOR_DARK_GRAY        "#404040"
+#define COLOR_MAROON           "#800000"
+#define COLOR_PURPLE           "#800080"
+#define COLOR_VIOLET           "#9933ff"
+#define COLOR_OLIVE            "#808000"
+#define COLOR_BROWN_ORANGE     "#824b28"
+#define COLOR_DARK_ORANGE      "#b95a00"
+#define COLOR_GRAY40           "#666666"
+#define COLOR_GRAY20           "#333333"
+#define COLOR_GRAY15           "#151515"
+#define COLOR_SEDONA           "#cc6600"
+#define COLOR_DARK_BROWN       "#917448"
+#define COLOR_BLUE             "#0000ff"
+#define COLOR_DEEP_SKY_BLUE    "#00e1ff"
+#define COLOR_LIME             "#00ff00"
+#define COLOR_CYAN             "#00ffff"
+#define COLOR_TEAL             "#33cccc"
+#define COLOR_RED              "#ff0000"
+#define COLOR_PINK             "#ff00ff"
+#define COLOR_PALE_PINK        "#bf89ba"
+#define COLOR_ORANGE           "#ff9900"
+#define COLOR_YELLOW           "#ffff00"
+#define COLOR_YELLOW_GRAY      "#c9a344"
+#define COLOR_PALE_YELLOW      "#c1bb7a"
+#define COLOR_WARM_YELLOW      "#b3863c"
+#define COLOR_GRAY             "#808080"
+#define COLOR_RED_GRAY         "#aa5f61"
+#define COLOR_BROWN            "#b19664"
+#define COLOR_GREEN_GRAY       "#8daf6a"
+#define COLOR_DARK_GREEN_GRAY  "#54654c"
+#define COLOR_BLUE_GRAY        "#6a97b0"
+#define COLOR_DARK_BLUE_GRAY   "#3e4855"
+#define COLOR_SURGERY_BLUE     "#e0f2f6"
+#define COLOR_SUN              "#ec8b2f"
+#define COLOR_PURPLE_GRAY      "#a2819e"
+#define COLOR_BLUE_LIGHT       "#33ccff"
+#define COLOR_RED_LIGHT        "#ff3333"
+#define COLOR_BEIGE            "#ceb689"
+#define COLOR_BABY_BLUE        "#89cff0"
+#define COLOR_PALE_GREEN_GRAY  "#aed18b"
+#define COLOR_PALE_RED_GRAY    "#cc9090"
+#define COLOR_PALE_PURPLE_GRAY "#bda2ba"
+#define COLOR_PALE_BLUE_GRAY   "#8bbbd5"
+#define COLOR_LUMINOL          "#66ffff"
+#define COLOR_SILVER           "#c0c0c0"
+#define COLOR_GRAY80           "#cccccc"
+#define COLOR_OFF_WHITE        "#eeeeee"
+#define COLOR_WHITE            "#ffffff"
+#define COLOR_GOLD             "#ffcc33"
+#define COLOR_CLOSET_GOLD      "#6d6133"
+#define COLOR_NT_RED           "#9d2300"
+#define COLOR_BOTTLE_GREEN     "#1f6b4f"
+#define COLOR_PALE_BTL_GREEN   "#57967f"
+#define COLOR_GUNMETAL         "#545c68"
+#define COLOR_WALL_GUNMETAL    "#353a42"
+#define COLOR_STEEL            "#a8b0b2"
+#define COLOR_MUZZLE_FLASH     "#ffffb2"
+#define COLOR_CHESTNUT         "#996633"
+#define COLOR_BEASTY_BROWN     "#663300"
+#define COLOR_WHEAT            "#ffff99"
+#define COLOR_CYAN_BLUE        "#3366cc"
+#define COLOR_LIGHT_CYAN       "#66ccff"
+#define COLOR_PAKISTAN_GREEN   "#006600"
+#define COLOR_HULL             "#436b8e"
+#define COLOR_AMBER            "#ffbf00"
+#define COLOR_COMMAND_BLUE     "#46698c"
+#define COLOR_SKY_BLUE         "#5ca1cc"
+#define COLOR_PALE_ORANGE      "#b88a3b"
+#define COLOR_CIVIE_GREEN      "#b7f27d"
```

Comment:
As above, might not match

Reply:
It does, checked via hexadecimal thingy.